tests/spicy/types/unit/field-max-size-fail.spicy

```diff
@@ -0,0 +1,10 @@
+# @TEST-EXEC-FAIL: spicyc -j %INPUT >>output 2>&1
```

Comment:
With this branch rebased onto `master` no diagnostic is triggered here as of 28733d55e388050bffd2d702f257f5812c1db20b.

Reply:
Still needs a rebase on the fix for #791.